CONTRIBUTING.md

```diff
@@ -203,6 +203,45 @@ $ docker run --rm -it -v /path/where/you/have/cloned/the/repository\:/src:ro -v
 This configuration should generally cover  your needs. For more details about other configurations, see [documentation in the SyTest repo](https://github.com/matrix-org/sytest/blob/develop/docker/README.md).
 
 
+## Run the integration tests ([Complement](https://github.com/matrix-org/complement)).
+
+[Complement](https://github.com/matrix-org/complement) is a suite of black box tests that can be run on any homeserver implementation. It can also be thought of as end-to-end (e2e) tests.
+
+It's often nice to develop on Synapse and write Complement tests at the same time.
+Here is how to run your local Synapse checkout against your local Complement checkout.
+
+(checkout [`complement`](https://github.com/matrix-org/complement) alongside your `synapse` checkout)
+```sh
+COMPLEMENT_DIR=../complement ./scripts-dev/complement.sh
+```
+
+To run a specific test file, you can adjust [`scripts-dev/complement.sh`](https://github.com/matrix-org/synapse/blob/develop/scripts-dev/complement.sh) to use the `-run MyTest` syntax. The name comes from the name structure in your Complement tests. If you're unsure of the name, you can do a full run and copy it from the test output:
```

Comment:
It would be nice to adjust the `complement.sh` script to have this as its own parameter to passthrough so the script itself doesn't have to be modified.

Another PR ⏩

Reply:
It looks like this is already possible! [`scripts-dev/complement.sh#L63-L64`](https://github.com/matrix-org/synapse/blob/b7186c6e8ddacc328ae2c155162b36291a3c2b79/scripts-dev/complement.sh#L63-L64)

```
COMPLEMENT_DIR=../complement ./scripts-dev/complement.sh TestJumpToDateEndpoint
```

Reply:
Yeah this is already possible, can we update this paragraph to mention how to do this instead of recommending editing the file?

Reply:
Ahh, yes sorry. I updated the command snippets below but not this text 👍